samples/features/azure-arc-data-controller/README.md

```diff
@@ -0,0 +1,4 @@
+# Azure Arc Data Controller cluster
+
+## Samples Setup
+Follow the instrutions here: https://raw.githubusercontent.com/ananto-msft/sql-server-samples/master/samples/features/azure-arc-data-controller/deployment/kubeadm/ubuntu-single-node-vm/README.md
```

Comment:
Make it relative name. This has your branch name. Can you follow the other README format? Add a sentence describing the parent folder "This folder contains samples for Azure Arc". In the setup section, you can link to the "deployment" folder. This is sufficient since if you have README.md in a folder the git UI will automatically display it.

Reply:
Done.